CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - Introduce possibility of specifying policy order restrictions in their schemas. APIcast now shows a warning when those restrictions are not respected [#1088](https://github.com/3scale/APIcast/pull/1088), [THREESCALE-2896](https://issues.jboss.org/browse/THREESCALE-2896)
+- Fix typos on JWT claim policy jsonschema [THREESCALE-3046](https://issues.jboss.org/browse/THREESCALE-3046)
```

Comment:
I think we should move this to a "Fixed" sub-section.

Reply:
:man_facepalming: true